wdl2cwl/tests/wdl_files/bwa.wdl

```diff
@@ -0,0 +1,112 @@
+version 1.0
+
+# Copyright (c) 2017 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task Mem {
+    input {
+        File read1
+        File? read2
+        # BwaIndex bwaIndex
+        String outputPrefix
+        Boolean sixtyFour = false
+        Boolean usePostalt = false
+        Int sortMemoryPerThreadGb = 2
+        Int compressionLevel = 1
+
+        String? readgroup
+        Int? sortThreads
+
+        Int threads = 4
+        Int? memoryGb
+        # Int timeMinutes = 1 + ceil(size([read1, read2], "G") * 220 / threads)
+        # Contains bwa 0.7.17 bwakit 0.7.17.dev1 and samtools 1.10.
+        String dockerImage = "quay.io/biocontainers/mulled-v2-ad317f19f5881324e963f6a6d464d696a2825ab6:c59b7a73c87a9fe81737d5d628e10a3b5807f453-0"
+    }
+
+    # Samtools sort may block the pipe while it is writing data to disk.
+    # This can lead to cpu underutilization.
+    # 1 thread if threads is 1. For 2-4 threads 2 sort threads. 3 sort threads for 5-8 threads.
+    # Int estimatedSortThreads = if threads == 1 then 1 else 1 + ceil(threads / 4.0)
+    # Int totalSortThreads = 2 # select_first([sortThreads, estimatedSortThreads])
+    # BWA needs slightly more memory than the size of the index files (~10%). Add a margin for safety here.
+    # Int estimatedMemoryGb = 1 + ceil(size(bwaIndex.indexFiles, "G") * 1.2) + sortMemoryPerThreadGb * totalSortThreads
```

Comment:
There is not point in testing `select_first` with only one input. So please turn `estimatedMemoryGb` into an input of type `Int` like `memoryGb`